### PR TITLE
improve website readability

### PR DESCRIPTION
**Pull request by Roiocam**
## Improvement

- Cancel the hover animation of the card to avoid guiding users that the card is clickable.
- Explicitly guide the user, the links are clickable.
- Improve card style.
- Increase the width of content on the page
- Strongly distinguish between plain text and links
- Using the secondary light background

## Result

截屏2024-01-24 18 23 12

截屏2024-01-24 18 22 10

### CODE REVIEW

**Comment by samueleresca** [comment]
Thanks the changes looks good 😄 . Few notes:
- You will need to run the [manual publishing process](https://github.com/apache/incubator-pekko-site/wiki/Manual-Publishing) described in the read me
- Could you please check the changes work on a mobile layout

**Comment by Roiocam** [comment]
> * Could you please check the changes work on a mobile resolution

I just verified everything is ok on the mobile device, and working well on the responsive mode too.

File: content/assets/stylesheets/home.css [hunk 1]
```diff
@@ -179,16 +179,22 @@ h1 {
...
   cursor: default;
+  padding:24px;
+  color: #ff9100;
+  opacity: 0.8;
 }
 
-.row > ul > li:hover {
-  opacity: 0.8;
+.row > ul > li  a{
+  text-decoration: underline;
```
Comment (He-Pin) [inline content/assets/stylesheets/home.css:197]: cool.

File: content/assets/stylesheets/home.css [hunk 2]
```diff
@@ -200,3 +206,33 @@ h1 {
...
+    max-width: 400px;
+  }
+  .row ul li {
+    padding: 4px;
+  }
+
+  .row > ul > li {
+    padding: 16px 8px;
+  }
+}
```
Comment (He-Pin) [inline content/assets/stylesheets/home.css:238]: add a newline here

**Review by He-Pin — APPROVED** [review]
lgtm

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**